client/go/Dockerfile

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM golang:latest
+
+RUN apt-get update && \
+    apt-get -y install git unzip build-essential autoconf libtool protobuf-compiler libprotobuf-dev
+RUN go get google.golang.org/grpc
+RUN go get github.com/golang/protobuf/protoc-gen-go
+
+# Install Go OpenCV (to simplify postprocessing)
+RUN apt-get install -y sudo && \
+    git clone https://github.com/hybridgroup/gocv.git && \
+    cd gocv && \
+    make install
```

Comment:
this step fails behind proxy unless this line is added:
RUN echo "Acquire::http::Proxy \"$http_proxy\";" > /etc/apt/apt.conf.d/proxy.conf